coordinators/src/main/java/com/squareup/coordinators/Coordinators.java

```diff
@@ -48,6 +48,9 @@ public static void bind(View view, CoordinatorProvider provider) {
    * to the group.
    */
   public static void installBinder(ViewGroup viewGroup, final CoordinatorProvider provider) {
+    for (int i = 0; i < viewGroup.getChildCount(); ++i) {
```

Comment:
1. please pull `viewGroup.getChildCount()` into a variable so it's only called once.
2. this is a nit, but generally we prefer `i++` to `++i`